WordPress/src/main/res/layout/my_site_fragment.xml

```diff
@@ -372,6 +372,39 @@
 
                     </com.google.android.material.card.MaterialCardView>
 
+                    <!--Plan-->
```

Comment:
- moved "Plan" section to the top 
- introduced "Jetpack" label
- introduced "Jetpack Settings" row